packages/headless/src/controllers/search-parameter-manager/headless-search-parameter-manager.test.ts

```diff
@@ -108,6 +108,20 @@ describe('search parameter manager', () => {
     });
   });
 
+  describe('#state.parameters.lq', () => {
+    it('when the parameter is not the default, it is included', () => {
+      engine.state.advancedSearchQueries.lq = 'abc';
+      engine.state.advancedSearchQueries.defaultFilters.lq = 'def';
+      expect(manager.state.parameters.lq).toBe('abc');
+    });
+
+    it('when the parameter is the default, it is not included', () => {
+      engine.state.advancedSearchQueries.lq = 'abc';
+      engine.state.advancedSearchQueries.defaultFilters.lq = 'abc';
+      expect('lq' in manager.state.parameters).toBe(false);
+    });
+  });
```

Comment:
Hey @tcarrio, let's remove support for `lq` from the search parameter manager and serializers. Two reasons for this:

1. While `lq` is a search parameter, this controller is focused on creating urls. Since the lq is typically a long string, it wouldn't make for a nice url. The lq is also user-specific, and shouldn't be shared with others.
2. The `lq` is on it's way out at Coveo, in favour of the new [case assist api](https://platform.cloud.coveo.com/docs?urls.primaryName=Customer%20Service#/Configurations) (aka customer service api). I'd prefer to scope the changes to just the advanced queries slice.


Some additional context on point 2:

The next version of the search api does not support `lq`. When headless migrates, we will have to deprecate `lq` support here. As long as headless is on the current search api version, I think it's ok to add `lq` support.

Reply:
Hey @samisayegh :wave: This sounds fine. We are hoping to move to the Case Assist controller whenever that is made available, but in the meantime we still need a case deflection flow. For that, we would replicate the functionality of Case Assist which requires the `lq` parameter.

When that's deprecated and Headless has support for Case Assist, that would definitely be the preferred way of interacting with it.

So on all counts, I agree.

I will remove `lq` from this controller. Let me know if there are any other changes that should be made :+1:

Reply:
Inclusion of the `lq` parameter has been removed from this controller. The tests have been updated to reflect that `lq` is not included in the output.

Reply:
Thanks for the changes 👍